XpringKit/DefaultXpringClient.swift

```diff
@@ -103,7 +103,8 @@ extension DefaultXpringClient: XpringClientDecorator {
   /// - Throws: An error if there was a problem communicating with the XRP Ledger.
   /// - Returns: The index of the latest validated ledger.
   public func getLatestValidatedLedgerSequence() throws -> UInt32 {
-    throw XRPLedgerError.unimplemented
```

Comment:
Why did you get rid of this?

Reply:
I've been implementing parts of this interface a method at a time to make it easy for reviewers to check my code (rather than having to check a whole class at once). 

In order to stub these methods before implementing them, I just threw an `unimplemented` exception. Now that this method is implemented, I can remove this exception and actually return a value.

Reply:
Oh I see thanks.